pkg/ddc/jindo/transform.go

```diff
@@ -663,11 +668,22 @@ func (e *JindoEngine) transformLabels(runtime *datav1alpha1.JindoRuntime, value
 	return nil
 }
 
+func (e *JindoEngine) transformNetworkMode(runtime *datav1alpha1.JindoRuntime, value *Jindo) (err error) {
+	// to set hostnetwork
+	if len(runtime.Spec.NetworkMode) == 0 || runtime.Spec.NetworkMode == NETWORKMODE_HOST {
```

Comment:
Would it be better to use `switch case` mode instead of `if else`

Reply:
thx, done